libqtile/widget/bluetooth.py

```diff
@@ -42,11 +42,14 @@ def _catch_dbus_error(msg):
     """Decorator to catch DBusErrors and log a message."""
 
     def _wrapper(func):
-        async def f(self):
+        async def f(self, *args, **kwargs):
             try:
-                await func(self)
+                await func(self, *args, **kwargs)
             except DBusError:
                 logger.warning(msg, self.name)
+            except BlockingIOError:
+                # Resource is temporarily unavailable
```

Comment:
Let's use a `logger.debug` here?

Reply:
Yes.

There are still some scenarios that aren't being caught (usually when laptop has been asleep and then resumed) so I need to catch those too and test some more.